stacks_queues/stack.py

```diff
@@ -21,18 +23,23 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if not self.store:
+            raise StackEmptyException
+        return self.store.remove_last()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        if self.store.empty():
+            return True
+        else:
+            return False
```

Comment:
Try to avoid the anti-pattern of testing a condition, and then returning the value that's the same results as the condition. Prefer
```suggestion
        return self.store.empty()
```